pkg/azurekeyvault/client/azure_keyvault_service.go

```diff
@@ -35,7 +35,7 @@ const (
 type Service interface {
 	GetSecret(secret *akvs.AzureKeyVault) (string, error)
 	GetKey(secret *akvs.AzureKeyVault) (string, error)
-	GetCertificate(secret *akvs.AzureKeyVault, exportPrivateKey bool) (*Certificate, error)
+	GetCertificate(secret *akvs.AzureKeyVault, exportPrivateKey bool, ensureServerFirst bool) (*Certificate, error)
```

Comment:
Introduce a option struct, as mentioned earlier?

Reply:
Yes, I agree, will make the changes.